reverse_engineering/helpers/parseConnectionString.js

```diff
@@ -4,11 +4,22 @@ const { URL } = require('url');
 const mssqlPrefix = 'mssql://';
 const sqlserverPrefix = 'jdbc:sqlserver://';
 
-// example: mssql://username:password@host:1433/DatabaseName
+/**
+ * Parse MS SQLServer connection URL.
+ *
+ * Example: mssql://username:password@host:1433/DatabaseName
+ *
+ * Name instance example: mssql://username:password@host\instance/DatabaseName
```

Comment:
I'm not sure whether that is a real example `host\instance/DatabaseName`
Moreover, the lib itself [has removed](https://github.com/tediousjs/node-mssql/blob/ea475b08b0ebc960a4b5ef9f83a89d3aacaef201/CHANGELOG.txt#L168C56-L168C111) the support of such URL.
Could you please share where did you get that format?